application/src/main/java/org/togetherjava/tjbot/DatabaseListener.java

```diff
@@ -12,11 +12,40 @@
 
 import java.util.Optional;
 
+/**
+ * Implementation of an example command to illustrate how to use a database.
+ * <p>
+ * The implemented commands are {@code !dbput} and {@code !dbget}. They act like some sort of simple
+ * {@code Map<String, String>}, allowing the user to store and retrieve key-value pairs from the
+ * database.
+ * <p>
+ * For example:
+ *
+ * <pre>
+ * {@code
+ * !dbput hello Hello World!
+ * // TJ-Bot: Saved under 'hello'.
+ * !dbget hello
+ * // TJ-Bot: Saved message: Hello World!
+ * }
+ * </pre>
+ */
 public final class DatabaseListener extends ListenerAdapter {
+    /**
+     * Logger for this class.
+     */
     private static final Logger logger = LoggerFactory.getLogger(DatabaseListener.class);
```

Comment:
I don't think that comment adds anything.

Reply:
```java
/**
 * This class represents a cat
 */
class Cat {
}
```